lookout/style/format/tests/test_quality_report.py

```diff
@@ -99,6 +110,34 @@ def test_no_model(self):
                         n_files=10, model_path=empty_model
                     )
 
+    def test_push_ananlyzer(self):
+        """Test push event to analyzer."""
+        try:
+            analyzer = "lookout.style.format.quality_report"
+            config_json = '{\"style.format.analyzer.QualityReportAnalyzer\": ' \
+                          '{\"model_path\": \"%s\", \"aggregate\":true}}' % self.model_path
+            self.test_analyzer = TestAnalyzer(port=self.port, db=self.db.name, fs=self.fs.name,
+                                              analyzer=analyzer)
+            self.test_analyzer.__enter__()
+            launch_server("push", "HEAD^^^^^^^^", "HEAD^^^^^^^", port=self.port,
```

Comment:
And here. Are you sure you cannot use hashes? They will be much less error-prone